src/SAWScript/Crucible/LLVM/Override.hs

```diff
@@ -1938,6 +2101,165 @@ storePointsToValue opts cc env tyenv nameEnv base_mem maybe_cond ptr val maybe_i
         Crucible.doConditionalWriteOperation sym base_mem cond store_op
     Nothing -> store_op base_mem
 
+-- | Like 'storePointsToValue', but specifically geared towards the needs
+-- of fields within bitfields. In particular, this performs all of the
+-- necessary bit-twiddling on the LHS (a bitfield) to store the RHS value in
+-- the right place in the bitfield.
+storePointsToBitfieldValue ::
+  ( ?memOpts :: Crucible.MemOptions
+  , ?w4EvalTactic :: W4EvalTactic
+  , Crucible.HasPtrWidth (Crucible.ArchWidth arch)
+  , Crucible.HasLLVMAnn Sym
+  ) =>
+  Options ->
+  LLVMCrucibleContext arch ->
+  Map AllocIndex (LLVMPtr (Crucible.ArchWidth arch)) ->
+  Map AllocIndex (MS.AllocSpec (LLVM arch)) ->
+  Map AllocIndex (MS.TypeName (LLVM arch)) ->
+  Crucible.MemImpl Sym ->
+  LLVMPtr (Crucible.ArchWidth arch) ->
+  BitfieldIndex ->
+  SetupValue (LLVM arch) ->
+  IO (Crucible.MemImpl Sym)
+storePointsToBitfieldValue opts cc env tyenv nameEnv base_mem ptr bfIndex val = do
+  let sym = cc ^. ccBackend
+
+  let alignment = Crucible.noAlignment -- default to byte alignment (FIXME)
+
+  smt_array_memory_model_enabled <- W4.getOpt
+    =<< W4.getOptionSetting enableSMTArrayMemoryModel (W4.getConfiguration sym)
+
+  -- Unlike in storePointsToValue, we compute the MemTy/StorageType not from
+  -- the RHS value, but from the BitfieldIndex. This is because we need to
+  -- load the entire bitfield, which can be larger than the size of the RHS
+  -- value.
+  let memTy = biBitfieldType bfIndex
+  storTy <- Crucible.toStorableType memTy
+
+  case val of
+    SetupTerm _tm
+      -- TODO RGS: File an issue about this
+      | smt_array_memory_model_enabled ->
+        fail $ unlines
+          [ "llvm_points_to_bitfield currently does not work in"
+          , "combination with the enable_smt_array_memory_model option."
+          ]
```

Comment:
As the code here indicates, `llvm_points_to_bitfield` doesn't work with `enable_smt_array_memory_model` at present. I should file an issue to track this.

Reply:
I've opened #1540 and mentioned it in comments in the places you'd expect.